scripts/validate_changelog.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/python
+
+import argparse
+import logging
+import re
+import subprocess
+import sys
+from collections import defaultdict
+from typing import Any, Dict, Tuple, Union
+
+try:
+    import yaml
+except ImportError:
+    pass
```

Comment:
If this is just to defeat pylint check the note on the pre-commit config, otherwise I'm not sure what it's for.

```suggestion
import yaml
```